pango-cairo-sys/src/lib.rs

```diff
@@ -0,0 +1,87 @@
+// This file was generated by gir (0fe730d) from gir-files (062ad3b)
+// DO NOT EDIT
+
+#![allow(non_camel_case_types, non_upper_case_globals)]
+
+extern crate libc;
+#[macro_use] extern crate bitflags;
+extern crate glib_sys as glib;
+extern crate pango_sys as pango;
+extern crate cairo_sys as cairo;
+
+#[allow(unused_imports)]
+use libc::{c_int, c_char, c_uchar, c_float, c_uint, c_double,
+    c_short, c_ushort, c_long, c_ulong,
+    c_void, size_t, ssize_t, intptr_t, uintptr_t, time_t, FILE};
+
+#[allow(unused_imports)]
+use glib::{gboolean, gconstpointer, gpointer, GType, Volatile};
+
+// Callbacks
+pub type PangoCairoShapeRendererFunc = Option<unsafe extern "C" fn(*mut cairo::cairo_t, *mut pango::PangoAttrShape, gboolean, gpointer)>;
+
+// Classes
+//#[repr(C)]
+//pub struct PangoCairoFcFontMap {
+//    pub parent_instance: /*Metadata mismatch*/[c:type mismatch PangoFcFontMap != PangoCairoFcFontMap of FcFontMap],
```

Comment:
Seems gir-files fix not much help :(

Reply:
That's not the only problem apparently. This gir file is a mess.